compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -284,7 +283,7 @@ void buildAMDAIETransformPassPipeline(OpPassManager &pm) {
 
   if (clUsePipeline != AIEPassPipeline::PackPipeline) {
     auto &modulePassManager = pm.nest<ModuleOp>();
-    addMLIRAIRAIELoweringPasses(modulePassManager);
+      addMLIRAIRAIELoweringPasses(modulePassManager);
```

Comment:
I think this file should not be changed?

Reply:
Oh yes, that was a quirk of my rebasing. Will change it back

Reply:
done